test/paginator_test.exs

```diff
@@ -9,8 +9,14 @@ defmodule PaginatorTest do
   setup :create_data
 
   test "paginates forward", %{
-    payments: {p1, p2, p3, p4, p5, p6, p7, p8, p9, p10, p11, p12}
+    payments: {p1, p2, p3, p4, p5, p6, p7, p8, p9, p10, p11, p12} = payments
   } do
+    payments
+    |> Tuple.to_list()
+    |> Enum.with_index()
+    |> Enum.map(fn {p, i} -> {NaiveDateTime.to_iso8601(p.charged_at), p.id, "p#{i + 1}"} end)
+    |> Enum.sort()
```

Comment:
Is the result of this expression used?

Reply:
🙈 I was debugging something and forget to remove it. Doing that now.